core/profile_urls.py

```diff
@@ -1,8 +1,9 @@
 from django.urls import path
 
-from core.views import SavedPostListView, UserPostListView
+from core.views import MyCommentsListView, SavedPostListView, UserPostListView
 
 profile_urlpatterns = [
     path("profile/saved-post-list/", SavedPostListView.as_view(), name="saved_posts"),
+    path("profile/my-comments-list", MyCommentsListView.as_view(), name="my_comments"),
```

Comment:
```suggestion
    path("profile/comments-list/", UserCommentsListView.as_view(), name="user_comments"),
```
This naming is more readable, I think. Also added missing trailing slash.
Please update other places.